recipe/build.sh

```diff
@@ -5,7 +5,7 @@ set -x
 export TMPDIR=$SRC_DIR/temp_files
 mkdir -p $TMPDIR
 
-if [[ ${target_platform} == linux-ppc64le ]]; then
+if [[ ${target_platform} != *-64 ]]; then
   # https://github.com/libgd/libgd/issues/278
   export CFLAGS="$CFLAGS -ffp-contract=off"
 elif [[ ${target_platform} == linux-32 ]]; then
```

Comment:
`linux-32` is obsolete already and can be removed

Reply:
yes.else case is dead anyway ... as linux-32 is != *-64 ... on the other hand we keep less diff with CF